links.json

```diff
@@ -1,80 +1,79 @@
 {
-    "$schema": "http://json-schema.org/draft-06/hyper-schema#",
-    "$id": "http://json-schema.org/draft-06/links#",
+    "$schema": "http://json-schema.org/draft-07-wip/hyper-schema#",
+    "$id": "http://json-schema.org/draft-07-wip/links#",
     "title": "Link Description Object",
-    "type": "object",
-    "required": [ "href" ],
-    "properties": {
-        "anchor": {
-            "type": "string",
-            "format": "uri-template"
-        },
-        "anchorPointer": {
-            "type": "string",
-            "anyOf": [
-                { "format": "json-pointer" },
-                { "pattern": "^[0-9]" }
-            ]
-        },
-        "rel": {
-            "type": "string"
-        },
-        "href": {
-            "type": "string",
-            "format": "uri-template"
-        },
-        "templatePointers": {
+    "allOf": [
+        { "required": [ "rel", "href" ] },
+        { "$ref": "#/definitions/noRequiredFields" }
+    ],
+    "definitions": {
+        "noRequiredFields": {
```

Comment:
This `noRequiredFields` name seems a bit odd, especially as it's used in "hyper-schema-output.json". Maybe "ldo" would be more meaningful but on the other hand it does not convey the fact that "required" is absent on purpose so it wouldn't be a real LDO. I have no better idea to suggest at the moment, just mentioning in case somebody has an idea...

Reply:
Yeah, I'm not particularly fond of the name but nothing else was coming to mind so I went with it for now.  It's at least descriptive.  But hopefully someone will have a better idea :-)